CMakeLists.txt

```diff
@@ -303,9 +339,9 @@ IF (USE_SHA1DC)
 ELSEIF (WIN32 AND NOT MINGW)
 	ADD_DEFINITIONS(-DGIT_SHA1_WIN32)
 	FILE(GLOB SRC_SHA1 src/hash/hash_win32.c)
-ELSEIF (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+ELSEIF (TLS_BACKEND MATCHES "SecureTransport")
```

Comment:
IIRC CMake original behavior was to always substitute variables in some cases, `IF` being one of them.

I'll switch to `STREQUAL` though, as `TLS_BACKEND` is supposed to be an internal option.